cli/ballerina-cli/src/main/resources/cli-help/ballerina-tool-use.help

```diff
@@ -0,0 +1,15 @@
+NAME
+       bal-tool-use - Mark a version of a tool as the active version
```

Comment:
we have used the pattern `ballerina-build`. To be consistent, lets use the same pattern here as well. E.g. `ballerina-tool-use`